src/app/applications/datatarget/fiware/fiware-edit/fiware-edit.component.html

```diff
@@ -85,11 +85,32 @@
 
   <div class="row mb-2">
     <div class="form-group mt-3 col-12">
-        <label class="form-label" for="authorizationHeader">{{'QUESTION.GIVE-DATATARGET-AUTHORIZATIONHEADER' | translate}}</label>
-        <input type="text"
-          class="form-control" id="authorizationHeader" name="authorizationHeader" required
-          [(ngModel)]="datatarget.authorizationHeader"
-          [ngClass]="{'is-invalid' : formFailedSubmit && errorFields.includes('authorizationHeader'), 'is-valid' : formFailedSubmit && !errorFields.includes('authorizationHeader')}">
+      <label class="form-label" for="authorizationHeader">{{'QUESTION.GIVE-DATATARGET-AUTHORIZATIONHEADER' | translate}}</label>
+      <input type="text"
```

Comment:
Fields are missing a placeholder

![image](https://user-images.githubusercontent.com/92869496/190352489-00c43e8e-4d01-49d0-b3de-e1274a2af519.png)

Reply:
Any suggestion about placeholders? There is no one exact value that makes sense to me as it may vary depending on the identity manager you use.

Reply:
Token: `https://os2iot/connect/token`
Client id: Client's navn
Client secret: Client's kode

It looks like you've added them in the code but they don't show up on the page. Can you reproduce it?

Reply:
Done